src/site_assessment/common_functions.jl

```diff
@@ -287,6 +287,22 @@ function output_geojson(
     return nothing
 end
 
+"""
+    output_geojson(df::DataFrame, destination_path::String)::Nothing
+
+Writes out GeoJSON file to a target directory. Output file will be located at location:
+`destination_path`.
+
+# Arguments
+- `df` : DataFrame intended for writing to geojson file.
+- `destination_path` : File path to write geojson file to.
+"""
+function output_geojson(df::DataFrame, destination_path::String)::Nothing
+    GDF.write(destination_path, df; crs=GI.crs(first(df.geometry)))
+
+    return nothing
+end
```

Comment:
Very minor note:

Base Julia functions and FileIO standardize to `function_name(file_path, data)`

You can see here GeoDataFrames have also standardized too this order of arguments.

Best we conform as well to reduce unexpected surprises?